libantworld/render_target_input.cc

```diff
@@ -0,0 +1,38 @@
+#include "render_target_input.h"
+
+// Libantworld includes
+#include "render_target.h"
+
+//----------------------------------------------------------------------------
+// BoBRobotics::AntWorld::RenderTargetInput
+//----------------------------------------------------------------------------
+namespace BoBRobotics
+{
+namespace AntWorld
+{
+cv::Size RenderTargetInput::getOutputSize() const
+{
+    return cv::Size(m_RenderTarget.getWidth(), m_RenderTarget.getHeight());
```

Comment:
Maybe it makes sense just to have ``RenderTarget`` use ``cv::Size``?

Reply:
I think there is one as it uses it for loading textures but I don't reaaaally want to introduce too much coupling between libantworld and opencv

Reply:
We are using OpenCV basically everywhere though... but it's up to you.